src/icmp/Icmp4.cc

```diff
@@ -132,6 +132,11 @@ Icmp4::SendEcho(Ip::Address &to, int opcode, const char *payload, int len)
     icmp->icmp_cksum = CheckSum((unsigned short *) icmp, icmp_pktsize);
 
     to.getAddrInfo(S);
```

Comment:
According to getAddrInfo(), we are supposed to specify AF_INET if we "care". Based on the checks below (and this overall context), we do care!

```suggestion
    to.getAddrInfo(S, AF_INET);
```